go.mod

```diff
@@ -69,9 +69,9 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gorm.io/gorm v1.25.2
 	k8s.io/api v0.26.1
-	k8s.io/apiextensions-apiserver v0.26.0
```

Comment:
It comes from `go mod tidy` for checking the dependencies compatibility, a minor version bump should be ok.